contrib/debian/bitcoin-qt.desktop

```diff
@@ -3,6 +3,7 @@ Encoding=UTF-8
 Name=Bitcoin
 Comment=Bitcoin P2P Cryptocurrency
```

Comment:
I checked some programs and the default seems to be imperative style, so maybe change to:

Connect to the Bitcoin P2P Network